solr/test-framework/src/java/org/apache/solr/cloud/AbstractFullDistribZkTestBase.java

```diff
@@ -257,16 +260,27 @@ public void distribSetUp() throws Exception {
 
     if (isSSLMode()) {
       System.clearProperty(URL_SCHEME);
-      try (ZkStateReader zkStateReader = new ZkStateReader(zkServer.getZkAddress(),
-          AbstractZkTestCase.TIMEOUT, AbstractZkTestCase.TIMEOUT)) {
+      try (ZkStateReader zkStateReader =
```

Comment:
an aside: perfect spot to use "var" -- the variable's name makes the type redundant.  And it's shorter.